deploy/src/main/resources/production-swarm/README.md

```diff
@@ -25,6 +25,10 @@ The `environment.sh` file supports configurable property settings necessary for
 	export CAF_ELASTIC_NUMBER_OF_SHARDS=5
 	## The number of replica shards (copies) that each primary shard should have.
 	export CAF_ELASTIC_NUMBER_OF_REPLICAS=1
+        ## A comma separated list of Elasticsearch HOST value.
+        export CAF_ELASTIC_HOST_VALUES=192.168.56.10
+        ## The REST port of the Elasticsearch server listens on.
+        export CAF_ELASTIC_PORT_VALUE=9200
```

Comment:
- Indenting - the rest of this file uses tabs
- Do we need to have both options listed here?  If so then it is not clear that they are alternatives.